src/Notes/CompleteSetup.php

```diff
@@ -34,9 +35,9 @@ public function get_note_name(): string {
 	}
 
 	/**
-	 * Possibly add the note
+	 * Check if the note should be added each day.
 	 */
-	public function possibly_add_note(): void {
+	public function run(): void {
```

Comment:
How about we delegate the responsibility of actually adding the notes to another service or job instead of having each note add itself? Meaning that the note classes like `CompleteSetup` and `ContactInformation` just define the notes but the task of adding the note be handled by another service.

Reply:
Yeah that could work well. I guess in this case we might not really need `RunDaily` interface and `cron` handler. Is that what you were thinking?

Reply:
Yeah, what I was thinking was that another central class implements those interfaces and manages all other notes in the system instead of each note being responsible for adding itself.